TUTORIAL.md

```diff
@@ -0,0 +1,223 @@
+
+The exact steps for your upgrade are likely to differ, so the process needs to be adapted to your environment.
+
+## Overview
+
+Review the expedited upgrade documentation on TechDocs: 
+- [Preparing for Expedited Upgrade](https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-enterprise-software/layer7-api-management/api-gateway/10-0/install-configure-upgrade/upgrade-the-gateway/upgrade-an-appliance-gateway/prep-for-expedited-upgrade.html)
+- [Automating with Ansible](https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-enterprise-software/layer7-api-management/api-gateway/10-0/install-configure-upgrade/upgrade-the-gateway/upgrade-an-appliance-gateway/automated-expedited-upgrade/automating-with-ansible.html)
+- [Automated Expedited Upgrade Procedure](https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-enterprise-software/layer7-api-management/api-gateway/10-0/install-configure-upgrade/upgrade-the-gateway/upgrade-an-appliance-gateway/automated-expedited-upgrade.html)
+- [Post Expedited Upgrade Tasks](https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-enterprise-software/layer7-api-management/api-gateway/10-0/install-configure-upgrade/upgrade-the-gateway/upgrade-an-appliance-gateway/post-upgrade-tasks.html)
+
+Outline of the upgrade steps to follow:
+
+![Steps for in-place upgrade](https://techdocs.broadcom.com/content/dam/broadcom/techdocs/us/en/assets/docops/gateway/ansibleAutoReimage.png/_jcr_content/renditions/original)
+
+## Scenario
+
+For context, the upgrade scenario we will use is an expedited in-place upgrade for a Virtual Appliance Gateway cluster, which will re-use existing VM resources:
+- Starting Point: a Gateway v9.4 cluster with 3 nodes, Gateway MySQL v5.7 database is configured with failover, and OAuth Toolkit v4.3.1 is installed
+
+- Upgraded Gateways: same as above but with v10.0 Gateways which run a MySQL v8 database.
+
+## Setup Inventory 
+
+Copy the [inventories/sample](/inventories/sample) directory to `inventories/tutorial`.
+
+Update the `hosts` file in this new directory with your Gateway deployment information. Guidelines are [here](inventories/README.md).
+
+Note: for this scenario, the upgraded and source Gateway hostnames are identical!
+
+For example:
+```yaml
+---
+all:
+  hosts:
+  children:
+    gateway:
+      children:
+        gateway_mysql:
+          children:
+            gateway_primary_db:
+              hosts:
+                source-db-gw.placeholder.com:
+                  src: source-db-gw.placeholder.com
+            gateway_failover_db:
+              hosts:
+                source-failover-db-gw.placeholder.com:
+                  src: source-failover-db-gw.placeholder.com
+        gateway_processing_node:
+          hosts:
+            source-processing-gw.placeholder.com:
+                src: source-processing-gw.placeholder.com
+```
+
+## Setup Passwords
+
+Enter Gateway passwords in the inventory's `/group_vars/vault` file. 
```

Comment:
"/group_vars/vault" should be /group_vars/all/vault

Reply:
fixed